pnpm/dev/pd.js

```diff
@@ -2,6 +2,7 @@
 const fs = require('fs')
```

Comment:
There's a shebang above this line, but sadly adding `--enable-source-maps` is insufficient to make this work, since pnpm's bin stub invokes this script using Node already, not Bash, so the shebang is ignored.